indigo_app/js/components/DocumentTOCView.vue

```diff
@@ -1,69 +1,29 @@
 <template>
   <div class="toc-controller-wrapper">
-    <div class="input-group mb-2">
-      <input type="text"
-             class="form-control form-control-sm"
-             placeholder="Search by title"
-             v-model="titleQuery"
-      >
-      <div class="input-group-prepend">
-        <button class="btn btn-sm btn-secondary"
-                style="width: 25px"
-                type="button"
-                @click="clearTitleQuery"
-                :disabled="!titleQuery"
-        >
-          <i class="fas fa-times"></i>
-        </button>
-      </div>
-    </div>
-    <div class="d-flex mb-2">
-      <button @click="expandAllTOCItems"
-              type="button"
-              class="btn btn-primary btn-sm mr-1"
-      >
-        Expand All
-      </button>
-      <button @click="collapseAllTOCItems"
-              type="button"
-              class="btn btn-primary btn-sm">
-        Collapse All
-      </button>
-    </div>
-    <TOCController
-        :items="roots"
-        :title-query="titleQuery"
-        ref="tocController"
-        @on-title-click="onTitleClick"
+    <la-table-of-contents-controller
+        ref="la-toc-controller"
+        :items.prop="roots"
+        :title-filter="titleQuery"
+        expand-all-btn-classes="btn btn-primary btn-sm mr-1"
+        collapse-all-btn-classes="btn btn-primary btn-sm mr-1"
+        search-filter-input-classes="form-control form-control-sm"
+        title-filter-placeholder="Search by title"
```

Comment:
Can we now rely on the default placeholder, since that has been updated?

Reply:
done